python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -758,6 +761,74 @@ def convert_to_cudf(cp_arrays):
             ddf = self.renumber_map.unrenumber(ddf, "second")
 
         return ddf
+    
+    def select_random_vertices(self, random_state=None, num_vertices=None):
```

Comment:
Could you add explicit tests for this function?

Reply:
I added both SG and MG tests